src/sanity_side.rs

```diff
@@ -57,12 +63,25 @@ fn main() -> anyhow::Result<()> {
 }
 
 lazy_static! {
-    static ref DEV_MAP: RwLock<HashMap<String, rdma::ibv::IbvCtx>> = RwLock::new(HashMap::new());
-    static ref QP_MAP: RwLock<Vec<rdma::ibv::IbvQp>> = RwLock::new(vec![]);
-    static ref PD_MAP: RwLock<Vec<rdma::ibv::IbvPd>> = RwLock::new(vec![]);
-    static ref CQ_MAP: RwLock<Vec<(rdma::ibv::IbvCq, rdma::ibv::IbvEventChannel)>> =
-        RwLock::new(vec![]);
-    static ref MR_MAP: RwLock<Vec<Pin<Vec<u8>>>> = RwLock::new(vec![]);
+    static ref RDMA_MAP: Arc<RwLock<HashMap<String, Rdma>>> = Arc::new(RwLock::new(HashMap::new()));
```

Comment:
Is it necessary to use tokio lock?

Reply:
Yes, we need tokio lock in `local_recv` call, otherwise we can't spawn the task.